processor/batchprocessor/batch_processor.go

```diff
@@ -182,6 +183,13 @@ func (bp *batchProcessor) ConsumeMetrics(_ context.Context, md pdata.Metrics) er
 	return nil
 }
 
+// ConsumeLogs implements LogsProcessor
+func (bp *batchProcessor) ConsumeLogs(_ context.Context, ld pdata.Logs) error {
+	// First thing is convert into a different internal format
```

Comment:
The comment does not seem to reflect what the line that comes after it does.

Reply:
Fixed.